calico/getting-started/kubernetes/openshift/installation.mdx

```diff
@@ -25,7 +25,7 @@ to install {{prodname}}.
 
 - Ensure that you have a [RedHat account](https://cloud.redhat.com/). A RedHat account is required to obtain the pull secret necessary to provision an OpenShift cluster.
 
-- Ensure that you have installed the OpenShift installer **v4.3 or later** and OpenShift command line interface from [cloud.redhat.com](https://cloud.redhat.com/openshift/install/aws/installer-provisioned).
+- Ensure that you have installed the OpenShift installer and OpenShift command line interface from [cloud.redhat.com](https://cloud.redhat.com/openshift/install/aws/installer-provisioned).
```

Comment:
should we call out `supported Openshift installer and Openshift command line ...` since we are removing `4.3 or later` ?

Reply:
Yes, good one. Fixed.